### PR TITLE
Make domElementGetter optional

### PR DESCRIPTION
**Pull request by mellis481**

### CODE REVIEW

File: src/single-spa-preact.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ const defaultOpts = {
 	// required opts
 	preact: null,
 	rootComponent: null,
+	
+  // optional opts
```
Comment (internettrans) [inline src/single-spa-preact.js:6]: This file has a mixture of tabs and spaces. It was originally tabs - I'm okay with changing to spaces but we should only have one kind of indentation.

**Review by internettrans — COMMENTED** [review]
Did you test this locally before creating the pull request?

File: src/single-spa-preact.js (hunk 1 above)
Comment (mellis481) [reply src/single-spa-preact.js:6]: Oof. Sorry. Not sure what happened when I pasted. Should be good now.

Comment (mellis481) [reply src/single-spa-preact.js:6]: Oof. Sorry. Not sure what happened when I pasted. Should be good now.

**Review by internettrans — APPROVED** [review]
Thanks

**Pull request merged**